pages/index.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-    <div>
+    <main>
         <banner/>
         <explore/>
         <categories/>
         <lovedLists/>
-        Gamers Music
-    </div>
+        <footer>Gamers Music</footer>
```

Comment:
remove this since Mare is doing footer and we almost done with that, footer needs to be called in default.vue below <nuxt /> to show on all pages in the app